mattermost-10.2.advisories.yaml

```diff
@@ -579,6 +579,11 @@ advisories:
             componentType: apk
             componentLocation: /.PKGINFO
             scanner: grype
+      - timestamp: 2024-12-04T14:41:29Z
+        type: false-positive-determination
+        data:
+          type: component-vulnerability-mismatch
+          note: 'This vulnerability relates to v7.x of mattermost, which is several releases old. The componentVersion is also being flagged incorrectly here by some scanners. A bug has been filed upstream against Syft, and the maintainers have confirmed it''s a scanner issue. See: https://github.com/anchore/syft/issues/2980.'
```

Comment:
The component version flagged by Syft is `10.2.0-r0`, right? This match is to the APK (as seen in the detection event above), not a Go module — so I don't think that Syft issue is relevant here.

This match seems to be from an overly broad CPE configuration on the NVD record at https://nvd.nist.gov/vuln/detail/CVE-2022-4045, where where are no version constraints, so all versions of Mattermost are getting matched.